### PR TITLE
sasview saying unknown distribution option 'test_requires'

### PR DESCRIPTION
**Pull request by pkienzle**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def find_version(package):
         'OpenCL': ["pyopencl"],
         'CUDA': ["pycuda"],
     },
-    build_requires=['setuptools'],
-    test_requires=['pytest'],
-    cmdclass = {'test': PyTest},
+    #setup_requires=['setuptools'],
```
Comment (rozyczko) [inline setup.py:78]: The reason you removed `setup_requires` is that `setuptools` is being imported by this script anyway and we wouldn't be able to run it without `setuptools` installed, I guess?

File: setup.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def find_version(package):
...
+    tests_require=['pytest'],
```
Comment (rozyczko) [inline setup.py:79]: Hmm... running `python setup.py test` should show this typo earlier, are you running unit tests during Travis builds?

File: setup.py (hunk 1 above)
Comment (pkienzle) [reply setup.py:78]: I removed it due to [this report](https://github.com/pytest-dev/pytest-xdist/issues/136) that pip doesn't work with `setup_requires`.  I haven't verified that this is an issue either way for sasmodels, so I'm going with less is more.

File: setup.py (hunk 2 above)
Comment (pkienzle) [reply setup.py:79]: Tests run during travis builds use the environment that we set up, which includes pytest.

**Pull request merged**